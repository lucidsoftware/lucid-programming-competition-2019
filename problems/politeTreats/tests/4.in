88071 shocking vomit-inducing bogeyman sugar
22542 frightening vomit-inducing crypt treat
20501 eerie poisonous-looking ghost mix
92188 shocking poisonous-looking tomb chocolate
-4068 shocking brain melting death mix
65567 spooky double chocolate haunted fudge
16432 alarming fireball grim taffy
38966 frightening iced death pure sugar
88118 chilling mysterious demon sourstick
59453 horrendous vomit-inducing skull treat
81985 spine-chilling double chocolate witch frosting
43079 chilling iced monster gummy
98376 horrifying crushed poltergeist pie
8267 spooky green mausoleum butter
67670 eerie fireball skeleton cottoncandy
49239 horrifying triple fudge mist donut
71767 horrifying crushed howl butter
28768 hair-raising elemental potion doughnut
6241 hair-raising elemental tombstone treat
98407 spooky green vampire caramel apple
59504 hairy orange-flavored afterlife treat
118 spine-chilling iced mist caramel apple
98423 shocking double chocolate tombstone caramel apple
32887 frightening green tombstone candy bar
96379 horrifying definitely-poisonous coffin bars
24702 shocking magical corpse caramel apple
34952 shocking vomit-inducing scarecrow frosting
67723 unnerving iced ghoul cottoncandy
34956 eerie crushed monster mix
98447 horrifying green phantasm doughnut
82065 horrendous crushed werewolf gummy
82066 frightening quadruple rasberry potion bars
59537 shocking possibly-poisonous scarecrow chocolate
57492 bloodcurdling foul-looking wraith creme de la crumb
-1898 horrifying mysterious poltergeist liquorice
14488 hair-raising brain melting troll donut
16554 bloodcurdling iced nightmare cake
43184 hair-raising brain melting warlock cake
61620 alarming glucose exploding cobweb creme de la crumb
67767 bloodcurdling elemental spirit mix
6329 eerie triple fudge demon mix
71868 shocking iced potion cake
65740 alarming iced skull pumpkin
10457 creepy brain melting enchantment caramel apple
45274 hair-raising quadruple rasberry vampire creme de la crumb
73946 bloodcurdling double chocolate monster treat
-1827 frightening possibly-poisonous afterlife candy bar
28894 shocking possibly-poisonous nightmare doughnut
96481 unnerving crushed haunted chocolate
43233 spine-chilling crushed fangs pure sugar
53475 spooky magical ghost pastry
-3866 spine-chilling oozing devil gummy
53489 unnerving double chocolate mummy liquorice
59634 horrifying elemental skull fudge
51445 hair-raising fireball spirit doughnut
22774 unnerving glucose exploding troll gummy
28924 spine-chilling vomit-inducing jack-o-lantern gummy
84225 shocking mysterious mausoleum cottoncandy
35081 chilling iced gravestone donut
94476 horrendous magical monster frosting
78100 bloodcurdling possibly-poisonous cadaver taffy
82205 chilling elemental spider fudge
86302 frightening iced coffin gummy
289 frightening orange-flavored ogre fudge
33060 shocking triple fudge spell candy bar
51500 spine-chilling diabetic nighmarish hocus pocus donut
86317 alarming green grave candy bar
35119 spooky crushed werewolf sourstick
43312 horrendous diabetic nighmarish corpse pure sugar
39220 bloodcurdling glucose exploding tombstone cottoncandy
59705 horrendous orange-flavored fangs cake
61764 horrendous definitely-poisonous skeleton mix
53573 spine-chilling triple fudge ghoul chocolate
63817 chilling iced bogeyman butter
24906 hairy brain melting cadaver pie
16717 frightening vomit-inducing devil gummy
61778 bloodcurdling glucose exploding corpse butter
35154 chilling double chocolate monster donut
74067 creepy triple fudge halloween pumpkin
18770 horrifying green moonlight candy bar
76123 spine-chilling double chocolate troll mix
49502 creepy crushed zombie mix
8553 frightening poisonous-looking haunted fudge
90474 spooky diabetic nighmarish demon cake
37243 intimidating iced tomb sucker
33151 unnerving quadruple rasberry cadaver butter
4480 chilling iced mausoleum pure sugar
16771 creepy definitely-poisonous skull bars
53636 chilling triple fudge wraith creme de la crumb
8583 shocking foul-looking poltergeist sugar
92552 bloodcurdling possibly-poisonous monster liquorice
43401 spooky foul-looking coffin pure sugar
24972 shocking oozing coffin doughnut
10637 horrendous iced genie creme de la crumb
33166 bloodcurdling possibly-poisonous hocus pocus mix
49550 horrifying brain melting witchcraft taffy
80270 spooky iced spirit cottoncandy
22940 alarming definitely-poisonous tombstone fudge
61856 horrendous fireball pumpkin pure sucrose
27045 hairy definitely-poisonous fairy sucker
37289 frightening iced ghoul caramel apple
39338 spine-chilling quadruple rasberry troll chocolate
55729 creepy quadruple rasberry phantasm cake
43459 hair-raising foul-looking poltergeist liquorice
6608 horrendous double chocolate mausoleum butter
82388 unnerving mysterious ghost sucker
10711 eerie magical grave pure sugar
59864 hairy brain melting mausoleum chocolate
57819 eerie fireball goblin donut
27099 alarming oozing crypt donut
74212 hair-raising diabetic nighmarish goblin chocolate
59879 hair-raising possibly-poisonous ogre candy bar
491 unnerving crushed ogre cottoncandy
59886 bloodcurdling foul-looking spirit pure sugar
2543 spooky double chocolate phantom creme de la crumb
20996 horrifying crushed corpse liquorice
10762 hair-raising glucose exploding werewolf pie
2575 horrifying green grim liquorice
27159 bloodcurdling diabetic nighmarish cadaver candy bar
4633 chilling foul-looking witch sourstick
2589 horrendous oozing spell frosting
84512 hair-raising vomit-inducing poltergeist frosting
51744 spooky diabetic nighmarish wraith cottoncandy
70182 shocking oozing fangs cottoncandy
18984 eerie fireball grave butter
98859 shocking iced tombstone caramel apple
41520 eerie fireball spell butter
39473 chilling diabetic nighmarish fangs frosting
82490 eerie possibly-poisonous cadaver candy bar
84538 horrifying crushed tarantula pure sugar
84541 horrifying glucose exploding goblin mix
31295 bloodcurdling crushed devil donut
96842 unnerving brain melting fangs pumpkin
86606 creepy elemental death chocolate
603 bloodcurdling iced genie gummy
59999 spine-chilling oozing halloween bars
17005 frightening orange-flavored poltergeist candy bar
55921 spine-chilling poisonous-looking coffin cake
29299 eerie definitely-poisonous nightmare sourstick
41590 spooky iced genie fudge
29304 hair-raising diabetic nighmarish grim sourstick
82555 creepy double chocolate spell frosting
82558 spine-chilling green goblin liquorice
-1408 hairy vomit-inducing scarecrow mix
12930 frightening triple fudge howl cottoncandy
12933 frightening double chocolate mummy creme de la crumb
68229 horrendous foul-looking ghoul caramel apple
57994 hair-raising magical afterlife doughnut
652 hair-raising poisonous-looking tarantula cottoncandy
90776 eerie vomit-inducing ghoul frosting
96925 creepy crushed nightmare doughnut
12960 horrendous elemental monster taffy
82594 unnerving oozing halloween bars
47781 intimidating quadruple rasberry bogeyman pure sugar
84647 creepy brain melting devil caramel apple
33447 shocking quadruple rasberry witch pumpkin
19112 hair-raising orange-flavored scarecrow pie
58024 spine-chilling fireball werewolf cottoncandy
88747 hair-raising double chocolate witch pastry
10926 alarming poisonous-looking grim donut
6831 eerie definitely-poisonous genie caramel apple
39601 eerie glucose exploding wraith pure sucrose
27327 intimidating poisonous-looking ogre sourstick
8898 spine-chilling magical skull sugar
47816 eerie oozing werewolf candy bar
8907 hairy quadruple rasberry hocus pocus pure sucrose
6861 hairy glucose exploding cemetery donut
56014 creepy possibly-poisonous mausoleum cottoncandy
88783 horrendous green skull pastry
74448 creepy iced death treat
60119 spooky diabetic nighmarish haunted donut
41688 creepy orange-flavored mist butter
72410 bloodcurdling foul-looking crypt pure sucrose
58085 shocking oozing mausoleum creme de la crumb
74475 frightening crushed haunted taffy
-3347 unnerving double chocolate ghost doughnut
27375 hair-raising double chocolate ogre creme de la crumb
92913 frightening mysterious skull liquorice
31484 bloodcurdling poisonous-looking tombstone butter
13069 hair-raising vomit-inducing demon taffy
70414 spine-chilling definitely-poisonous cobweb bars
86803 hairy foul-looking spell pure sugar
60183 eerie possibly-poisonous gravestone taffy
88859 hairy possibly-poisonous poltergeist pie
76575 hairy quadruple rasberry phantasm donut
68385 chilling poisonous-looking tombstone creme de la crumb
74534 intimidating oozing spirit creme de la crumb
95017 frightening mysterious witchcraft gummy
43818 spooky triple fudge death creme de la crumb
2859 bloodcurdling elemental pumpkin frosting
31536 frightening diabetic nighmarish grim caramel apple
84785 hairy triple fudge hocus pocus butter
6967 frightening iced fairy sourstick
31548 frightening oozing devil pumpkin
78656 alarming mysterious potion caramel apple
17217 spine-chilling orange-flavored werewolf pumpkin
68423 bloodcurdling vomit-inducing ogre sucker
19280 chilling fireball nightmare pastry
47953 spine-chilling mysterious wraith treat
76628 eerie definitely-poisonous warlock liquorice
39770 intimidating elemental fangs creme de la crumb
19292 creepy double chocolate cobweb pie
35678 creepy possibly-poisonous fairy cake
62316 spooky double chocolate phantasm sucker
56182 horrifying orange-flavored skeleton mix
50040 chilling double chocolate warlock pastry
11135 creepy foul-looking skeleton pure sugar
43906 hairy foul-looking goblin caramel apple
7044 intimidating iced enchantment mix
74634 spine-chilling possibly-poisonous monster candy bar
31638 spooky crushed mummy cake
95131 hairy magical demon pure sucrose
50081 eerie iced spell treat
99239 eerie glucose exploding ghost treat
48040 unnerving crushed haunted pure sucrose
41905 spooky elemental death cake
50100 unnerving mysterious werewolf gummy
19380 horrifying crushed mist creme de la crumb
64438 intimidating double chocolate fangs pumpkin
13240 horrifying elemental bogeyman cake
41916 eerie diabetic nighmarish gravestone fudge
52157 creepy crushed demon treat
3004 horrifying brain melting mummy pastry
-3137 horrifying oozing haunted sourstick
78785 horrifying brain melting phantasm sugar
5065 eerie green spider donut
3020 bloodcurdling definitely-poisonous howl taffy
27600 shocking elemental afterlife bars
52185 hair-raising quadruple rasberry spider caramel apple
74713 chilling fireball pumpkin gummy
99290 chilling definitely-poisonous mausoleum gummy
41949 chilling oozing fangs sucker
23519 horrendous triple fudge spider chocolate
95220 hairy quadruple rasberry warlock pastry
68597 frightening foul-looking grave pumpkin
91124 creepy quadruple rasberry zombie candy bar
37881 bloodcurdling elemental cobweb treat
80892 chilling crushed ghoul gummy
11262 intimidating brain melting tomb taffy
82950 alarming mysterious potion pie
1037 horrendous green witch taffy
72723 unnerving glucose exploding demon liquorice
25620 shocking brain melting potion mix
80915 horrendous foul-looking cobweb doughnut
56343 unnerving elemental werewolf taffy
72735 hairy poisonous-looking hocus pocus pure sucrose
17443 horrifying brain melting bogeyman taffy
42022 spooky brain melting tombstone treat
3111 horrifying crushed cemetery bars
76849 horrendous oozing nightmare cake
93239 spooky vomit-inducing jack-o-lantern sourstick
9275 creepy mysterious ghost caramel apple
25663 eerie brain melting crypt bars
70722 chilling crushed moonlight pure sugar
68680 creepy brain melting cobweb cottoncandy
91215 hairy mysterious spell cottoncandy
7253 eerie elemental spider pastry
33887 eerie foul-looking mist butter
46176 spine-chilling possibly-poisonous mummy caramel apple
13412 unnerving brain melting devil candy bar
85093 hairy fireball mist pumpkin
85099 bloodcurdling glucose exploding haunted butter
-914 creepy possibly-poisonous poltergeist doughnut
97394 horrendous triple fudge potion cottoncandy
46196 hair-raising possibly-poisonous gravestone pumpkin
1143 bloodcurdling green coffin frosting
81023 eerie poisonous-looking wraith frosting
95363 unnerving poisonous-looking cadaver sugar
97412 hairy brain melting mausoleum liquorice
52357 hair-raising iced poltergeist pure sugar
62601 bloodcurdling crushed moonlight doughnut
87180 chilling crushed spell liquorice
72853 unnerving possibly-poisonous goblin treat
25753 shocking elemental phantom treat
50341 spine-chilling magical corpse bars
44199 alarming magical death caramel apple
-855 alarming diabetic nighmarish haunted candy bar
21675 horrendous triple fudge tarantula mix
85164 eerie quadruple rasberry halloween pure sucrose
70832 intimidating magical halloween sucker
79025 creepy orange-flavored zombie caramel apple
3261 unnerving fireball howl pure sugar
9423 frightening diabetic nighmarish phantasm cake
21713 horrendous glucose exploding mummy frosting
85204 alarming orange-flavored troll cake
1240 alarming elemental grave pie
77016 spooky vomit-inducing fairy pumpkin
66780 horrifying quadruple rasberry haunted pumpkin
56553 alarming green vampire pie
77038 hair-raising oozing spider candy bar
54511 shocking iced tomb bars
31992 alarming glucose exploding bogeyman pumpkin
31993 eerie magical mummy sugar
5369 shocking poisonous-looking spell cake
11523 spooky iced tombstone doughnut
23818 eerie diabetic nighmarish warlock pure sucrose
17679 bloodcurdling crushed cadaver caramel apple
34064 spine-chilling green potion pastry
72975 horrendous foul-looking cobweb chocolate
62741 horrifying definitely-poisonous cobweb doughnut
91415 horrifying triple fudge scarecrow butter
70942 bloodcurdling oozing spirit treat
75041 chilling definitely-poisonous devil butter
85282 unnerving definitely-poisonous tomb pie
42279 unnerving triple fudge genie sucker
40233 eerie green fairy pastry
73006 eerie green ghost creme de la crumb
21817 alarming definitely-poisonous halloween gummy
99642 spine-chilling definitely-poisonous skull frosting
85312 spooky double chocolate death sugar
40266 unnerving oozing monster pie
-2735 alarming green cadaver creme de la crumb
81236 creepy possibly-poisonous haunted cake
11630 creepy fireball death fudge
11633 horrendous brain melting corpse pumpkin
56690 creepy elemental gravestone cottoncandy
68983 alarming possibly-poisonous corpse cake
7548 creepy crushed corpse butter
68989 hairy glucose exploding scarecrow cottoncandy
52615 hairy brain melting tombstone sucker
46471 chilling vomit-inducing tombstone pure sucrose
21903 intimidating oozing witchcraft sourstick
5521 frightening triple fudge goblin sucker
83350 bloodcurdling quadruple rasberry ghoul pastry
73110 shocking fireball genie cottoncandy
13720 frightening oozing cemetery donut
77210 horrifying poisonous-looking cobweb candy bar
23963 alarming iced cemetery caramel apple
15776 hairy poisonous-looking ghoul pastry
52646 frightening double chocolate vampire pastry
89517 hairy fireball skeleton candy bar
5550 eerie brain melting coffin mix
36277 horrendous triple fudge tombstone sugar
54710 alarming elemental devil chocolate
17847 intimidating poisonous-looking phantasm sucker
81340 hair-raising iced coffin mix
9662 hair-raising triple fudge fairy chocolate
97728 alarming vomit-inducing warlock taffy
58819 bloodcurdling green werewolf pure sugar
97734 spooky triple fudge mummy sucker
85448 eerie foul-looking demon butter
50633 alarming mysterious nightmare caramel apple
52680 spooky crushed haunted bars
81358 spine-chilling poisonous-looking ghoul fudge
77263 frightening green moonlight candy bar
9681 horrifying crushed gravestone gummy
40406 hairy vomit-inducing bogeyman cottoncandy
85462 hairy magical witch pie
48602 unnerving oozing demon gummy
73182 hairy oozing ogre sucker
26082 hairy definitely-poisonous mist pastry
87531 spooky glucose exploding corpse liquorice
83441 eerie iced witchcraft gummy
-521 unnerving oozing death butter
52729 unnerving triple fudge tomb pure sucrose
32250 chilling quadruple rasberry pumpkin mix
36347 frightening poisonous-looking ghoul sourstick
22026 creepy vomit-inducing tarantula sucker
99851 frightening magical enchantment pumpkin
65038 intimidating double chocolate werewolf candy bar
22043 shocking iced howl chocolate
19997 intimidating elemental tarantula candy bar
42529 alarming triple fudge jack-o-lantern liquorice
42531 hairy elemental death pure sugar
54823 spine-chilling elemental monster doughnut
87596 horrendous definitely-poisonous warlock frosting
13874 unnerving vomit-inducing devil pastry
20019 horrifying vomit-inducing nightmare sucker
87604 eerie magical enchantment butter
36414 bloodcurdling possibly-poisonous spell fudge
97859 spooky foul-looking genie pie
17991 alarming iced monster sugar
50765 hair-raising glucose exploding warlock doughnut
50766 alarming fireball zombie butter
34387 eerie green mist liquorice
87639 creepy triple fudge hocus pocus taffy
69209 horrendous vomit-inducing skull bars
46682 bloodcurdling magical wraith pastry
61021 shocking foul-looking spider sugar
69213 hairy poisonous-looking phantasm donut
-2462 spooky orange-flavored witch bars
95847 horrendous mysterious phantom sugar
52841 bloodcurdling poisonous-looking skeleton chocolate
46699 frightening mysterious grave cake
1647 intimidating definitely-poisonous mist pure sucrose
7797 eerie possibly-poisonous witchcraft sourstick
71288 shocking orange-flavored howl sucker
69243 eerie iced wraith creme de la crumb
73349 spooky definitely-poisonous witch doughnut
42629 hair-raising double chocolate corpse taffy
-374 hairy foul-looking cobweb bars
77456 bloodcurdling double chocolate afterlife pie
91798 horrendous double chocolate grim taffy
44698 creepy green grave pastry
46748 eerie glucose exploding phantasm donut
48806 creepy definitely-poisonous gravestone taffy
20138 unnerving diabetic nighmarish cadaver butter
7850 intimidating crushed mummy chocolate
59052 unnerving definitely-poisonous jack-o-lantern donut
18090 hairy quadruple rasberry wraith chocolate
9904 spine-chilling definitely-poisonous jack-o-lantern pastry
95922 bloodcurdling green crypt caramel apple
87737 horrifying iced spell cake
32446 alarming elemental haunted sugar
34497 spine-chilling elemental ghost fudge
-4414 shocking green fairy treat
36559 unnerving poisonous-looking afterlife creme de la crumb
38611 chilling crushed moonlight bars
50900 hairy diabetic nighmarish mausoleum sugar
93909 frightening triple fudge bogeyman doughnut
73432 bloodcurdling mysterious tarantula pumpkin
32473 shocking iced enchantment frosting
-4390 chilling iced skeleton sucker
98011 frightening glucose exploding warlock candy bar
57050 hairy vomit-inducing skull fudge
30429 eerie elemental wraith sugar
91876 spine-chilling fireball howl sourstick
48870 hair-raising mysterious skeleton pure sugar
14057 horrifying mysterious moonlight chocolate
63211 bloodcurdling fireball monster pure sugar
30444 unnerving glucose exploding grave pure sugar
55020 bloodcurdling brain melting skull mix
-274 shocking iced skeleton doughnut
55024 hair-raising glucose exploding witchcraft pumpkin
16113 spine-chilling glucose exploding zombie taffy
67314 horrifying orange-flavored fairy frosting
34553 shocking crushed troll doughnut
42751 intimidating possibly-poisonous moonlight gummy
-2304 hair-raising mysterious troll donut
34562 horrifying triple fudge monster doughnut
67337 horrifying definitely-poisonous haunted cottoncandy
-2290 shocking green bogeyman pastry
5910 unnerving elemental nightmare pure sucrose
71453 intimidating orange-flavored troll donut
46877 hairy fireball ghost fudge
30505 horrifying definitely-poisonous howl creme de la crumb
65322 alarming iced coffin butter
73515 chilling glucose exploding potion pure sucrose
59182 spine-chilling vomit-inducing hocus pocus donut
16181 horrendous quadruple rasberry werewolf sucker
26432 eerie quadruple rasberry afterlife frosting
46912 frightening vomit-inducing troll fudge
98113 hair-raising foul-looking ogre pure sucrose
69442 horrendous mysterious crypt treat
48965 spine-chilling green jack-o-lantern bars
48968 chilling poisonous-looking tombstone pie
91978 horrendous definitely-poisonous fangs creme de la crumb
71498 frightening crushed hocus pocus sugar
59212 spine-chilling triple fudge tarantula gummy
38736 hair-raising iced phantasm gummy
32593 shocking definitely-poisonous grim caramel apple
10071 shocking iced afterlife butter
61286 spooky iced death pie
30567 alarming foul-looking cadaver liquorice
22377 unnerving foul-looking mist cottoncandy
16234 creepy triple fudge coffin mix
34667 frightening magical hocus pocus sourstick
5995 hair-raising glucose exploding monster bars
77679 frightening magical phantom doughnut
87925 hair-raising glucose exploding grim bars
61307 creepy definitely-poisonous corpse pie
8062 hairy oozing death candy bar
18308 frightening iced phantom butter
40843 hair-raising orange-flavored tomb doughnut
57228 bloodcurdling green troll taffy
44941 hairy mysterious witch pure sucrose
92049 spine-chilling foul-looking tarantula sucker
59281 frightening green grim chocolate
49044 hair-raising foul-looking bogeyman sucker
16279 intimidating brain melting bogeyman cake
24471 horrifying iced cobweb caramel apple
24473 alarming magical genie gummy
77727 hair-raising iced gravestone pastry
81829 unnerving poisonous-looking mist cake
22442 bloodcurdling mysterious skeleton sucker
1965 alarming poisonous-looking fangs creme de la crumb
4028 spine-chilling triple fudge afterlife butter
90046 horrifying orange-flavored phantasm caramel apple
-4161 shocking magical fangs butter
61376 bloodcurdling quadruple rasberry ghoul liquorice
22463 horrendous orange-flavored mummy creme de la crumb
94144 spooky definitely-poisonous mausoleum creme de la crumb
73667 bloodcurdling mysterious spider pastry
61381 horrendous foul-looking ghost cake
75718 hairy iced scarecrow bars
90054 spine-chilling poisonous-looking ghoul sucker
12229 alarming brain melting halloween pure sugar
85962 hair-raising brain melting halloween pure sugar
6100 unnerving iced skeleton butter
71640 spine-chilling glucose exploding ghoul sucker
53214 unnerving elemental corpse pastry
34784 hairy possibly-poisonous werewolf sugar
34798 bloodcurdling magical ghost caramel apple
92145 creepy crushed poltergeist creme de la crumb
8178 chilling brain melting tomb treat
61426 hair-raising mysterious ghost pumpkin
26612 horrifying poisonous-looking enchantment pie
71670 hair-raising brain melting phantom liquorice
51195 shocking vomit-inducing mummy sucker
79868 chilling fireball ogre liquorice
