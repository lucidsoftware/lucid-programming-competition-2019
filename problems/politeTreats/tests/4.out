spine-chilling definitely-poisonous skull frosting
